wkcuber/downsampling.py

```diff
@@ -168,7 +168,10 @@ def downsample(
             wkw.Header.BLOCK_TYPE_LZ4HC if compress else wkw.Header.BLOCK_TYPE_RAW
         )
         ensure_wkw(
-            target_wkw_info, block_type=header_block_type, num_channels=num_channels
+            target_wkw_info,
```

Comment:
Nice stuff! I created the follow up issue we talked about: https://github.com/scalableminds/webknossos-cuber/issues/142